slm_porosity/heat_transfer.f90

```diff
@@ -458,9 +455,27 @@ SUBROUTINE user_stats ( u ,j_mn, startup_flag)
     INTEGER , INTENT (IN) :: startup_flag
     INTEGER , INTENT (IN) :: j_mn
     REAL (pr), DIMENSION (nwlt,1:n_var), INTENT (IN) :: u
+    REAL (pr) :: volume !volume of melting pool
+    REAL(pr), DIMENSION(1) :: width, length, depth !dimensions of melting pool
+    
+    IF (startup_flag.EQ.0) THEN
+      OPEN(12, file='melting_pool.txt', status='replace', action='write')
+      WRITE(12,*) '#### THIS IS DIMENSIONS OF MELTING POOL ####'
```

Comment:
remove this line. It does not provide any new information and consists only on grammar errors.